docs/docs/development/reference/management-commands.md

```diff
@@ -113,6 +113,7 @@ The command allows you to fully define the name of your project or application,
 
 * `-d DIR, --dir=DIR` - An optional destination directory
 * `--with-auth` - Adds an authentication application to newly created projects. See [Authentication](../../authentication) to learn more about this capability
+* `--database` - Configures the applications database. Currently MySql, Postgresql and Sqlite3 are supported. See [Database settings](../settings#database-settings) for more information.
```

Comment:
I think we can also explicitly mention the expected raw values here (`mysql/postgresql/sqlite3`).